### DISCUSSION TITLE
Creation of categories in upptimerc.yml

### DISCUSSION THREAD

**Discussion by mayank0202** [upptime/upptime#944] [General] [answered]
Can we create categories/subcategories in upptimerc.yml ? 
Currently its like 
```
 - name: Google
   url : https://google.com
```
But i want something like

```
- category-1:
   subcategory:
      - name: google
         url : https://google.com
```
Is it possible in upptime?

**Comment by AnandChowdhary** [comment, marked as answer]
Unfortunately this is not supported, but it's one of the most popular feature requests: https://github.com/orgs/upptime/discussions/121. Happy to accept a PR if you're interesting in contributing!
